bundles/org.palladiosimulator.dataflow.nodecharacteristics.tests/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: org.palladiosimulator.dataflow.nodecharacteristics.tests;singleton:=true
+Automatic-Module-Name: org.palladiosimulator.dataflow.nodecharacteristics.tests
+Bundle-Version: 1.0.0.qualifier
+Bundle-ClassPath: .
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Bundle-RequiredExecutionEnvironment: J2SE-1.5
```

Comment:
Change to `JavaSE-17`

Reply:
Should be resolved by the most recent commit